pkgs/applications/networking/mailreaders/himalaya/default.nix

```diff
@@ -10,37 +10,45 @@
 , libiconv
 , openssl
 , notmuch
+, withRusttlsTls ? true
+, withRusttlsNativeCerts ? withRusttlsTls
+, withNativeTls ? false
```

Comment:
I would much prefer to see `withNativeTLS`. I generally don’t like to lose case information unless there is a good reason (imperfect string function like `toCamelCase`).

Reply:
I would argue on this one, except if there is Nix guidelines I am not aware of. [Rust API Guidelines](https://rust-lang.github.io/api-guidelines/naming.html) use `UpperCamelCase` and it makes sense to me. You lose case information, but you gain readability. Imagine the IMAP ID sequence, `IMAPIDSeq` is way less understandable than `ImapIdSeq`.

> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn.

Reply:
Would 100% disagree that that’s more readable as acronyms/initialism have meaning in their casing, but that’s why I left a _comment_ instead of _request changes_.

Reply:
Package `xh` also uses `withNativeTls` in parameters.

Reply:
So I will leave like this then, except if there is special naming related to Nix!